src/main/java/com/metamx/datatypes/openrtb/Video.java

```diff
@@ -54,8 +54,8 @@ public Video(
       @JsonProperty("maxduration") Integer maxDuration,
       @JsonProperty("protocol") Integer protocol,
       @JsonProperty("protocols") List<Integer> protocols,
```

Comment:
since "protocol" is deprecated, do we want to remove it from our builder? It would just be put in a list as "protocols"